components/sup/src/manager/mod.rs

```diff
@@ -594,7 +594,26 @@ impl Manager {
             self.fs_cfg.clone(),
             self.organization.as_ref().map(|org| &**org),
         ) {
-            Ok(service) => service,
+            Ok(service) => {
+                outputln!(
+                    "pkg version {}/{}/{}/{}",
+                    service.pkg.ident.origin,
+                    service.pkg.ident.name,
+                    service
+                        .pkg
+                        .ident
+                        .version
+                        .as_ref()
+                        .unwrap_or(&"UNKNOWN".to_string()),
+                    service
+                        .pkg
+                        .ident
+                        .release
+                        .as_ref()
+                        .unwrap_or(&"UNKNOWN".to_string())
+                );
```

Comment:
I think you should be able to replace all this with
```rust
outputln!("pkg version {}", service.pkg.ident);
```
since `PkgIdent` implements `Display`.

Reply:
Thinking a bit more, In terms of unifying the log output more, it may be better to remove the `outputln!` call from line 585 and do it all down here:

```rust
outputln!("Starting {} ({})", spec.ident, service.pkg.ident);
```
That would give us a single line like this:
```
hab-sup(MR): Starting core/redis (core/redis/3.2.4/20170514150022)
```
rather than
```
hab-sup(MR): Starting core/redis
hab-sup(MR): pkg version core/redis/3.2.4/20170514150022
```